src/cli.js

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env node
```

Comment:
To help with review, I created this branch in the extension to demonstrate all of the changes made to this file: https://github.com/MetaMask/metamask-extension/compare/auto-changelog-migration-changes

The other files migrated are all unchanged.